src/main/java/nl/knaw/dans/layerstore/LayerManagerImpl.java

```diff
@@ -100,7 +103,11 @@ private void archive(Layer layer) {
 
     @Override
     public Layer getLayer(long id) {
-        if (stagingRoot.resolve(Long.toString(id)).toFile().exists() || archiveProvider.exists(Long.toString(id))) {
+        if (id == topLayer.getId()) {
+            // an empty layer would not be found in the staging root
```

Comment:
I don´t understand this comment. Wouldn´t there be an empty staging directory?

Returning the `topLayer` here is OK.

Reply:
A new layer in the staging directory is only created when something is added.